src/containers/EditPortfolioListingPage/EditPortfolioListingWizard/EditPortfolioListingItemsPanel/EditPortfolioListingFilesForm.js

```diff
@@ -117,16 +130,27 @@ const EditPortfolioListingFilesFormComponent = props => {
   const listingId = listing?.id;
   const listingState = listing?.attributes?.state;
   const isDraft = listingState === LISTING_STATE_DRAFT;
+  const draftButtonText = (onNavigateToVideos ? 'Next' : 'Publish');
```

Comment:
I was checking the code, and I wonder, is there an scenario where 'onNavigateToVideos' can be empty?